go/lambda-cron/lambda-cron.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/aws/aws-cdk-go/awscdk"
+	"github.com/aws/aws-cdk-go/awscdk/awsevents"
+	"github.com/aws/aws-cdk-go/awscdk/awseventstargets"
+	"github.com/aws/aws-cdk-go/awscdk/awslambda"
+	"github.com/aws/constructs-go/constructs/v3"
+	"github.com/aws/jsii-runtime-go"
+)
+
+type LambdaCronStackProps struct {
+	awscdk.StackProps
+}
+
+func NewLambdaCronStack(scope constructs.Construct, id string, props *LambdaCronStackProps) awscdk.Stack {
+	var sprops awscdk.StackProps
+	if props != nil {
+		sprops = props.StackProps
+	}
+	stack := awscdk.NewStack(scope, &id, &sprops)
+
+	// The code that defines your stack goes here
+
+	code, err := ioutil.ReadFile("lambda-handler.py")
+
+	if err != nil {
+		panic(fmt.Sprintf("Failed reading handler code: %s", err))
+	}
+
+	lambdaFn := awslambda.NewFunction(stack, jsii.String("Singleton"), &awslambda.FunctionProps{
+		Code: awslambda.NewInlineCode(jsii.String(string(code))),
```

Comment:
Why inline code and not an asset? Is that the same as the original sample?

Reply:
Yeah its the same in the original. But happy to change, this will also show a direct asset usage we still don't have 👍

Reply:
Yes, let's do it

Reply:
Done. @eladb